src/components/modals/CreateAntModal/CreateAntModal.tsx

```diff
@@ -380,60 +383,68 @@ function CreateAntModal({ show }: { show: boolean }) {
+                                            background:
+                                              editingField === row.attribute
+                                                ? 'white'
+                                                : 'transparent',
+                                            border: 'none',
+                                            borderRadius: '2px',
+                                            color:
+                                              editingField === row.attribute
+                                                ? 'black'
+                                                : 'white',
+                                            padding:
+                                              editingField === row.attribute
+                                                ? '10px '
+                                                : '10px 0px',
+                                            display: 'block',
+                                          }}
+                                          disabled={
+                                            editingField !== row.attribute
+                                          }
+                                          placeholder={`Enter a ${mapKeyToAttribute(
+                                            row.attribute,
+                                          )}`}
+                                          value={
+                                            editingField !== row.attribute
+                                              ? row.value
+                                              : modifiedValue
+                                          }
+                                          setValue={(e) => {
+                                            setModifiedValue(e);
+                                            row.value = e;
+                                          }}
+                                          validationPredicates={
+                                            row.attribute === 'owner' ||
+                                            row.attribute === 'controller' ||
+                                            row.attribute === 'targetID'
+                                              ? {
+                                                  [VALIDATION_INPUT_TYPES.ARWEAVE_ID]:
+                                                    (id: string) =>
+                                                      arweaveDataProvider.validateArweaveId(
+                                                        id,
+                                                      ),
+                                                }
+                                              : {}
+                                          }
+                                          maxLength={43}
+                                          setIsValid={(b: boolean) => {
+                                            row.isValid = b;
```

Comment:
what is this doing?

Reply:
its stubbing the setIsValid prop - can add logic to the validation input to make thus nullable

Reply:
updated